<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="a_DSB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="b_DSB">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="c_DSB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Value of a</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Value of b</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Value of c</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Here we will calculate the answers of a quadratic formula, in this form: axÂ²+bx+c = 0</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
